src/mmw/templates/registration/activation_complete.html

```diff
@@ -7,7 +7,7 @@
 <p>
     {% trans "Your account is now activated." %}
     {% if not user.is_authenticated %}
-    {% trans "You can log in." %}
+    {% trans "If you had work in progress, please switch back to that tab and login to save your work." %}
```

Comment:
I agree it is clunky, and I agree this is a big enough issue to be dealt with separately. The activation link opens in a new tab. If we recreate the the original state and save it, what happens to the original tab? Do we send a signal from the server indicating that the project has been updated elsewhere? We don't have the WebSocket architecture in place for something like that. Or we could tell them to only use the new tab and close the current one as soon as they sign up. But in case they want to continue working for a while before activating their account, they will no longer be able to do so.

Created #399 for this.